src/corecel/Assert.hh

```diff
@@ -286,10 +287,11 @@
             }                                                      \
         } while (0)
 #else
-#    define CELER_HIP_CALL(STATEMENT)    \
-        do                               \
-        {                                \
-            CELER_NOT_CONFIGURED("HIP"); \
+#    define CELER_HIP_CALL(STATEMENT)                      \
+        do                                                 \
+        {                                                  \
+            CELER_NOT_CONFIGURED("HIP");                   \
+            (void)sizeof(celeritas_device_runtime_api_h_); \
```

Comment:
Is this not needed for `CELER_CUDA_CALL`?

Reply:
Not usually, as NVCC I think implicitly includes this? Though I guess we could be calling this from a cc file too so maybe I'll add it